source/LibKuribo/modules/kxer/Loader.cxx

```diff
@@ -287,6 +282,19 @@ LoadResult Load(const LoadParam& param, LoadedKXE& out) {
   if (result != LoadResult::Success)
     return result;
 
+#if KURIBO_PLATFORM_WII
+  // Cache blocks are 32 bytes in size
+  auto *cacheBegin = reinterpret_cast<u8*>(
+      reinterpret_cast<u32>(pCode.get()) & ~0x1F);
+  auto *cacheEnd = reinterpret_cast<u8*>(
+      (reinterpret_cast<u32>(pCode.get()) + pHeader->code.file_size + 0x20) & ~0x1F);
```

Comment:
We'd want to add `0x1f`, as in the case that `file_size` is a multiple of 32, we don't need to cover the next cache block.

Reply:
You're right